idea/src/org/jetbrains/kotlin/idea/inspections/SortModifiersInspection.kt

```diff
@@ -47,7 +47,7 @@ class SortModifiersInspection : AbstractKotlinInspection(), CleanupLocalInspecti
 
                 val descriptor = holder.manager.createProblemDescriptor(
                     startElement,
-                    list,
+                    modifierElements.last(),
```

Comment:
I think it would be better to highlight all modifiers, not only the last one (but annotations should not be included)